### PR TITLE
[14.0][MIG] product_multi_image

### PR DESCRIPTION
**Pull request by hailangvn**
Depends on https://github.com/OCA/server-tools/pull/2083
This module is needed for connector_prestashop to not install full blow ecommerce module `website_sale`.
(https://github.com/OCA/product-attribute/pull/358#issuecomment-528880199)
This is based on https://github.com/OCA/product-attribute/pull/626

### CODE REVIEW

File: product_multi_image/hooks.py
Comment (MeetKD) [inline product_multi_image/hooks.py]: Openerp - > odoo

Comment (hailangvn) [reply product_multi_image/hooks.py]: Thanks for your reviewing. It has been updated.

**Review by leemannd — COMMENTED** [review]
I don't know this module enough, but the migration is currently failing on https://travis-ci.com/github/OCA/product-attribute/jobs/507440812#L1582
`test_restrict_one_image`

Are you planning to work on it or is help welcome on it?

**Comment by hailangvn** [comment]
I tried to fix it but not yet success, even with image.mixin. So any help is welcome. Thanks in advance.
The WIP branches are https://github.com/hailangvn/server-tools/tree/wip/image_mixin and https://github.com/hailangvn/product-attribute/tree/wip/image_mixin.

**Comment by hailangvn** [comment]
Let me check this and provide the plan.

**Comment by hailangvn** [comment]
Good day @leemannd, @rousseldenis, this PR has been updated. Thanks.

**Review by flotho — APPROVED** [review]
tested on runboat + code review

**Comment by rousseldenis** [comment]
@hailangvn Doesn't this need migration scripts ?

**Comment by hailangvn** [comment]
Good day @rousseldenis, if you are mentioning `image, image_medium, image_small` fields, we believe [openupgrade convert_image_attachments](https://github.com/OCA/OpenUpgrade/blob/0c4d9c944ec989d6ce837291782d00b8b8462654/addons/product/migrations/13.0.1.2/post-migration.py#L12) will take care of converting the images to the "new" format.

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@MeetKD @leemannd Could you update your review ? Thanks

**Review by leemannd — APPROVED** [review]

**Review by MeetKD — APPROVED** [review]

**Comment by flotho** [comment]
Hi @rousseldenis , how to identify the maintainer or  PSC member to progress in the merging of this awesome PR

**Comment by flotho** [comment]
Hi @rousseldenis @gurneyalex , the website tells us that you're maintainer of this repo https://odoo-community.org/psc-teams/product-27
Any chance to merge this PR ?

**Review by flotho — APPROVED** [review]

**Comment by flotho** [comment]
Hi @rousseldenis @gurneyalex @pedrobaeza 

This PR is marked ready to merge, could we expect a merge soon?
It will help a lot for OCA/connector-prestashop#147

**Comment by pedrobaeza** [comment]
Please rebase and fix pre-commit changes.

**Comment by flotho** [comment]
@hailangvn , any chance to rebase and check recommit messages

**Comment by hailangvn** [comment]
Thanks @pedrobaeza, @flotho . This has been rebased.

**Comment by flotho** [comment]
@pedrobaeza , this one is rebased and cleaned up.
Do we need a new approval ?

**Comment by flotho** [comment]
yihaaaa...... awesome !

**Pull request merged**
